uwblueprint/paramedics-react
# Add snackbars

## Overview
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

[Notion task](https://www.notion.so/uwblueprintexecs/on-entity-create-event-snackbar-4b64273fa7aa423d8504aada96547f28)

For further reference:
https://www.notion.so/uwblueprintexecs/Front-End-Best-Practices-ba6081af21f14de0bfe3f7d12a757e12#61d0324e30fb4745b8201af4d3137860

## Changes
- Set up a way to easily add snackbars
- Add styling for all snackbars
- Add snackbars for the following
    - Users (or team members) add/edit/delete
    - Hospitals add/edit/delete
    - Ambulances add/edit/delete
- Removed unused `ResourceSnackbars` component

## Testing
A snackbar should appear for each of these cases:
- Go to '/manage/members'
    - Add a team member
    - Edit a team member
    - Delete a team member

- Go to '/manage/hospitals'
    - Add a hospital
    - Edit a hospital
    - Delete a hospital

- Go to '/manage/ambulances'
    - Add an ambulance
    - Edit an ambulance
    - Delete an ambulance

## Screenshots
Resource management:
![image](https://user-images.githubusercontent.com/22457859/89960673-a807f200-dc0d-11ea-8d58-9403e331713b.png)
